.github/workflows/test.yml

```diff
@@ -34,11 +34,12 @@ jobs:
           python -W ignore -m pip install pytest-xdist[psutil]
 
       - name: Test with pytest
-        # We run 4 different suites here
+        # We run 5 different suites here
         # 1. Test just utils.datetime.py without pytz being installed
         # 2. Test just test_no_passport.py without passport dependencies being installed
         # 3. Test just test_rate_limiter.py without passport dependencies being installed
-        # 4. Test everything else
+        # 4. Test just test_meta.py, since otherwise race conditions happened
+        # 5. Test everything else
```

Comment:
I would really prefer to get to the root of this instead of leaving a mysterious comment that will gives us headaches at some point down the road :/

Reply:
I did not understand the root problem one bit, as I said, doing the check if telegram package is installed in the test two function calls before the setup.py install does did not yield it being there, and then setup.py thinks it is. I did not check however if using it in the class created venv folders works or not. I still would not know where the actual problem lies then however.